src/sas/qtgui/MainWindow/GuiManager.py

```diff
@@ -651,9 +651,7 @@ def addTriggers(self):
         self._workspace.actionAbout.triggered.connect(self.actionAbout)
         self._workspace.actionWelcomeWidget.triggered.connect(self.actionWelcome)
         self._workspace.actionCheck_for_update.triggered.connect(self.actionCheck_for_update)
-        self._workspace.actionLog_installed_packages.triggered.connect(self.actionLog_installed_packages)
-        self._workspace.actionLog_imported_packages.triggered.connect(self.actionLog_imported_packages)
-
```

Comment:
This can be just commented out, so we know we need to uncomment it when fixed.

Reply:
@rozyczko I was contemplating whether s better to remove or comment out. At the end I decided to remove and added note on commits to the issue https://github.com/SasView/sasview/issues/1987#issuecomment-1029373374.  But I am ok with commenting out if you think it is better.

Reply:
@wpotrzebowski I guess this is OK, since we can't really comment out the relevant code in the .ui file (it'll get overwritten on any change). 
Adding a new menu item is trivial and can be done later.
Let's have it removed then.